### ISSUE TITLE
path with **/* doesn't work on windows

### ISSUE THREAD

**Issue by tcoopman** [STRML/JSXHint#37]
All the paths in the examples with stars (*) don't work on windows. (I'm not sure if this is only on windows but can't test anything else at the moment).

As far as I can tell, the jshint source code expects a directory an not a glob pattern (see https://github.com/jshint/jshint/blob/master/src/cli.js#L415). I cannot find an example on the jshint page either with a glob pattern.

So I'm wondering if this is a bug or a mistake in the examples.

Running like this does work:

```
jsxhint app\jsx
```

**Comment by wadeharrell** [comment]
since there has been no traction at jshint, could JSXHint use glob internally before calling jshint?

**Comment by STRML** [comment]
Hi @wadeharrell , could you install JSXHint from the repository on windows and see if d2040866999fc07fc63b20c07bc544328c792ffd fixed it for you?

**Comment by wadeharrell** [comment]
looks to have done the trick!

[glob-all](https://www.npmjs.com/package/glob-all) looks pretty handy, shame to limit the added functionality to just windows.  With it you can do things like:
```
jsxhint ["scripts/**/*.jsx", "scripts/**/*.js", "!scripts/ads/badcode.js"]
```
tested that, it works!

Perhaps just call glob on line 75 & drop the windows stuff on 19 & 102?

**Comment by STRML** [comment]
Could, just wanted to avoid the small runtime penalty outside of Windows,
where in the usual case only filenames would be sent by the shell.

It's not really a big deal though, glob is very fast and in
SublimeLinter-jsxhint takes input from stdin, so what's a few ms between
friends?

**Comment by STRML** [comment]
This was fixed by d2040866999fc07fc63b20c07bc544328c792ffd and made available to all platforms in #49. It is now released as `0.12.0`.
